dataflowengineoss/src/main/scala/io/joern/dataflowengineoss/queryengine/AccessPathUsage.scala

```diff
@@ -39,6 +39,9 @@ object AccessPathUsage {
           val (base, tail) = toTrackedBaseAndAccessPathInternal(argOne.get)
           val path         = AccessPathHandling.memberAccessToPath(memberAccess, tail)
           (base, path)
+        case _ =>
+          logger.error(s"Missing handling for node type ${node.getClass}.")
```

Comment:
Since the process doesn't propagate an exception, I would choose to log a warning instead of an error
```suggestion
          logger.warn(s"Missing handling for node type ${node.getClass}.")
```

Reply:
done